src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/leaked-passwords/LeakedPasswordsLayout.tsx

```diff
@@ -117,63 +130,63 @@ export function LeakedPasswordsLayout(props: LeakedPasswordsLayoutProps) {
 
       if (!isComplete) {
         setSubscriberBreaches(updatedSubscriberBreaches);
+        setIsResolving(false);
+        return;
       }
-      // If all breaches in the step is fully resolved, take users to the next step
-      else {
-        router.push(nextStep.href);
-      }
-    } catch (error) {
-      console.error("Error updating breach status", error);
+      // If all breaches in the step are fully resolved,
+      // take users to the celebration view.
+      const doneSlug: LeakedPasswordsTypes =
+        props.type === "passwords"
+          ? "passwords-done"
+          : "security-questions-done";
+      router.push(`/redesign/user/dashboard/fix/leaked-passwords/${doneSlug}`);
+    } catch (_error) {
+      setIsResolving(false);
```

Comment:
Same Q as above, re: capture and send these errors to Sentry.

Reply:
[Same as above.](https://github.com/mozilla/blurts-server/pull/3791#discussion_r1404570173)